gpt4all-chat/qml/ChatView.qml

```diff
@@ -1047,10 +1047,9 @@ Rectangle {
                                                 }
                                             }
 
-                                            TextArea {
-                                                text: consolidatedSources.length + " " + qsTr("Sources")
+                                            Text {
+                                                text: qsTr("%1 Sources").arg(consolidatedSources.length)
                                                 padding: 0
-                                                readOnly: true
```

Comment:
Why'd you remove this?

Reply:
This was a TextField, which meant that clicking on it didn't expand the sources. I changed it to Text since it's not meant to be edited anyway.